schema/common.json

```diff
@@ -2,6 +2,22 @@
   "title": "Common",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "mode": {
+      "title": "System Mode",
+      "description": "Operating mode for the device. Defaults is 'active'.",
+      "enum": [
+        "initial",
+        "active",
+        "restart"
+      ]
+    },
+    "blobsets": {
+      "title": "System Blobsets",
+      "description": "Predefined system blobsets",
+      "enum": [
+        "_iot_config"
```

Comment:
So we can define _firmware_update etc in this enum, reference it with constants, but we aren't using it as a key filter (instead we have open regex on config_blobset.json:15 ) ?

Reply:
The enums are things that end up in constants in the code -- but I don't know how to make the schema match "this should be an enum or not start with _" -- There is theoretically a "propertyNames" field in the schema spec, but I can't get it to actually do anything so it's either not really implemented, or implemented in a way I don't expect!